databricks/README.md

```diff
@@ -198,96 +208,212 @@ logs:
         - type: multi_line
           name: new_log_start_with_date
           pattern: \d{2,4}[\-\/]\d{2,4}[\-\/]\d{2,4}.*" > /etc/datadog-agent/conf.d/spark.d/spark.yaml
-else
 
-  # CONFIGURE HOST TAGS FOR WORKERS
-  DD_TAGS="environment:\${DD_ENV}","databricks_cluster_id:\${DB_CLUSTER_ID}","databricks_cluster_name:\${DB_CLUSTER_NAME}","spark_host_ip:\${SPARK_LOCAL_IP}","spark_node:worker"
+    echo "Spark integration configured"
+  else
+    echo "Installing Datadog Agent on the worker."
 
-  # INSTALL THE LATEST DATADOG AGENT 7 ON DRIVER AND WORKER NODES
-  DD_INSTALL_ONLY=true DD_API_KEY=\$DD_API_KEY DD_HOST_TAGS=\$DD_TAGS bash -c "\$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh)"
-  
-  # CONFIGURE HOSTNAME EXPLICITLY IN datadog.yaml TO PREVENT AGENT FROM FAILING ON VERSION 7.40+
-  # SEE https://github.com/DataDog/datadog-agent/issues/14152 FOR CHANGE
-  hostname=\$(hostname | xargs)
-  echo "hostname: \$hostname" >> /etc/datadog-agent/datadog.yaml
-fi
-
-  # RESTARTING AGENT
+    # CONFIGURE HOST TAGS FOR WORKERS
+    DD_TAGS="environment:\${DD_ENV}","databricks_cluster_id:\${DB_CLUSTER_ID}","databricks_cluster_name:\${DB_CLUSTER_NAME}","spark_host_ip:\${SPARK_LOCAL_IP}","spark_node:worker"
+
+    # INSTALL THE LATEST DATADOG AGENT 7 ON DRIVER AND WORKER NODES
+    # CONFIGURE HOSTNAME EXPLICITLY IN datadog.yaml TO PREVENT AGENT FROM FAILING ON VERSION 7.40+
+    # SEE https://github.com/DataDog/datadog-agent/issues/14152 FOR CHANGE
+    DD_INSTALL_ONLY=true DD_API_KEY=\$DD_API_KEY DD_HOST_TAGS=\$DD_TAGS DD_HOSTNAME="\$(hostname | xargs)" bash -c "\$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_agent7.sh)"
+
+    # WAIT FOR DATADOG AGENT TO BE INSTALLED
+    while [ ! -f "/etc/datadog-agent/datadog.yaml" ]; do
+      echo "Waiting 2 seconds for the Datadog Agent to be installed"
+      sleep 2
+    done
+
+    echo "Datadog Agent is installed"
+  fi
+
+  # Avoid conflicts on port 6062
+  echo "process_config.expvar_port: 6063" >> /etc/datadog-agent/datadog.yaml
+
+  echo "Waiting 15 seconds"
   sleep 15
+
+  echo "Restart the agent"
   sudo service datadog-agent restart
 EOF
 
-# CLEANING UP
 chmod a+x /tmp/start_datadog.sh
 /tmp/start_datadog.sh >> /tmp/datadog_start.log 2>&1 & disown
-""", True)
+
 ```
+
 <!-- xxz tab xxx -->
 <!-- xxz tabs xxx -->
 
-#### Job cluster
-After creating the `datadog-install-job-driver-mode.sh` script, add the init script path in the [cluster configuration page][6].
+##### With a cluster-scoped init script 
 
-**Note**: Job clusters are monitored in `spark_driver_mode` with the Spark UI port.
+Cluster-scoped init scripts are init scripts defined in a cluster configuration. Cluster-scoped init scripts apply to both clusters you create and those created to run jobs.
 
+Use the Databricks UI to edit the cluster to run the init script:
```

Comment:
I wonder if it's worth adding the the navigation route to the UI basically Workspaces -> Shared or user specific folder.

Or even just mention Workspaces could be enough to give an idea where to go.

Reply:
We can add that!

Reply:
added